docroot/modules/custom/mass_content/src/Field/RelatedNodes.php

```diff
@@ -130,15 +130,6 @@ protected function links() {
       }
     }
 
-    $types = $this->getSetting('types');
```

Comment:
@weitzman Removing because I don't think we need this query, which just loads all nodes by type, and it ultimately triggered the issue of the top binder nav not loading. This also seems redundant when we are limiting the query by ID or URI in the other getQuery() calls. It's likely a slight performance drain as well.